examples/UI.Note-sample.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="/sap/opu/odata4/sap/ui_nte_demo_m_o4/srvd/sap/ui_nte_demo_m/0001/$metadata">
+    <edmx:Include Namespace="com.sap.gateway.srvd.ui_nte_demo_m.v0001" Alias="service" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Namespace="Org.OData.Core.V1" Alias="Core" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="UI.examples" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <Annotations Target="service.C_NTE_DEMO_MType">
+        <Annotation Term="UI.Facets">
+          <Collection>
+            <Record Type="UI.ReferenceFacet">
+              <PropertyValue Property="Target" AnnotationPath="_NoteBasic/@UI.Note" />
+            </Record>
+          </Collection>
+        </Annotation>
+      </Annotations>
+      <Annotations Target="service.C_NoteBasicTPType">
+        <Annotation Term="UI.Note">
+          <Record>
+            <PropertyValue Property="Title" Path="NoteBasicTitle">
+              <Annotation Term="UI.Hidden" Path="_NoteBasicType/NoteBasicHideTitle" />
+            </PropertyValue>
```

Comment:
Does UI5 already support this? So far we seem to only have
- annotations directly within annotations (ApplicableTerms annotated on a Term)
- annotations within annotation records (ApplicableTerms annotated on a ComplexType)

This seems to be the first time we have annotations within an annotation property value.

Reply:
> @thomas-kaltenbach, can UI5 consume this annotation?

I cannot say it for sure but I guess it should be no problem to read the annotation with the MetadataModel.